languages/bash.scm

```diff
@@ -459,16 +467,20 @@
 ; in the context of a (test_command); it does work in other contexts
 ; See https://github.com/tweag/topiary/pull/155#issuecomment-1364143677
 (binary_expression
-   left: _ @append_space
-   right: _ @prepend_space
+  left: _ @append_space
+  right: _ @prepend_space
 )
 
 ;; Case Statements
 
 ; Indentation block between the "in" and the "esac"
 (case_statement
   .
-  "case" . _ .  "in" @append_hardline @append_indent_start
+  "case"
+  .
```

Comment:
Question: is it typical for a `.` to be on its own line? Or should we rather treat it like a delimiter?

Reply:
I also personally prefer it to act as a delimiter. However, in Tree-sitter's example documentation, I think it's always written on its own line. Probably for the case of when you have a query like:

```scheme
(
  ((some) (really) (complicated) (thing))
  .
  [(another) (really) (complicated) (thing)]
)
```